pkgs/development/python-modules/capstone/default.nix

```diff
@@ -1,21 +1,24 @@
-{ lib, stdenv
-, buildPythonPackage
-, fetchPypi
+{ buildPythonPackage
+, capstone
 , fetchpatch
+, fetchPypi
+, lib
 , setuptools
-, capstone
+, stdenv
 }:
 
 buildPythonPackage rec {
   pname = "capstone";
-  version = stdenv.lib.getVersion capstone;
+  version = lib.getVersion capstone;
 
   src = capstone.src;
   sourceRoot = "${capstone.name}/bindings/python";
 
   postPatch = ''
     ln -s ${capstone}/lib/libcapstone${stdenv.targetPlatform.extensions.sharedLibrary} prebuilt/
     ln -s ${capstone}/lib/libcapstone.a prebuilt/
+  '' + lib.optionalString stdenv.targetPlatform.isAarch64 ''
+    sed -i 's/manylinux1/manylinux2014/' setup.py
```

Comment:
I would prefer using substituteInPlace because it throws a warning when it no longer matches.

Reply:
Changed to `substituteInPlace`.